project/merge.go

```diff
@@ -37,7 +37,8 @@ func Merge(p *Project, bytes []byte) (map[string]*ServiceConfig, error) {
 	datas := make(rawServiceMap)
 	err := yaml.Unmarshal(bytes, &datas)
 	if err != nil {
-		logrus.Fatalf("Could not parse config for project %s : %v", p.Name, err)
+		logrus.Errorf("Could not parse config for project %s : %v", p.Name, err)
+		return nil, err
```

Comment:
It seems like this `Merge` should be private?  Is it really part of the public interface? It's currently only called from one place in the same package, and from my reading seems like it shouldn't be public.